5-infrastructure/modules/base-env/variables.tf

```diff
@@ -0,0 +1,359 @@
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "env" {
```

Comment:
We only need short code and we can map others internally like `dev={code: d, long: development}` for lookups

Reply:
shared vpc network name uses the middle form, dev as well
I can let Chris know to change naming convention in 3-networks

Reply:
yes we require user input as `dev` but keep mapping internally for dev == d == development